### PR TITLE
replaced def get_mocked_*(): with just def get_*(): in tests

### PR DESCRIPTION
**Pull request by jspaaks**
**List of related issues or pull requests**

Refs: #228

**Describe the changes made in this pull request**

- replaced `get_mocked_repo` with `get_repo`
- replaced `get_mocked_checker` with `get_checker` 

**Instructions to review the pull request**

### CODE REVIEW

File: tests/github/repo1/test_checker_no_args.py [hunk 1]
```diff
@@ -14,132 +14,132 @@ class TestCheckerNoArgs(Contract):
 
     def test_check_checklist(self, mocker):
         with mocker:
-            mocked_checker = get_mocked_checker()
+            mocked_checker = get_checker()
```
Comment (sverhoeven) [inline tests/github/repo1/test_checker_no_args.py:17]: Can the variable also be renamed to `checker`? 

Let's also remove `mocked_` from all variables set from a `get_checker()` or `get_repo()` method.

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Almost there lets get rid of more `mocked_` occurrences.

File: tests/github/repo1/test_checker_no_args.py (hunk 1 above)
Comment (jspaaks) [reply tests/github/repo1/test_checker_no_args.py:17]: done

**Review by sverhoeven — APPROVED** [review]
Nice, shorter clearer names everywhere.

**Pull request merged**